app/Console/Commands/OnlineControllerUpdate.php

```diff
@@ -55,13 +54,8 @@ public function __construct() {
      */
     public function handle() {
         $statsData = $this->getStatsData();
-        $last_update_log = ControllerLogUpdate::get()->first();
-        $last_update_log->delete();
-        $update_now = new ControllerLogUpdate;
-        $update_now->save();
-
+        $now = Carbon::now();
```

Comment:
This needs to be a timestamp...
![image](https://github.com/ZTL-ARTCC/Current_Website/assets/21279582/a48d2e93-047d-4594-97f6-f80fcec24245)

Reply:
Resolved and squashed into aeb04cb